specs/_features/eip7594/polynomial-commitments-sampling.md

```diff
@@ -481,12 +481,20 @@ def verify_cell_kzg_proof_batch_impl(row_commitments: Sequence[KZGCommitment],
                                      cosets_evals: Sequence[CosetEvals],
                                      proofs: Sequence[KZGProof]) -> bool:
     """
-    Verify a set of cells, given their corresponding proofs and their coordinates (row_index, column_index) in the blob
-    matrix. The i-th cell is in row row_indices[i] and in column column_indices[i].
-    The list of all commitments is provided in row_commitments_bytes.
+    Helper: Verify that a set of cells belong to their corresponding commitment.
 
-    This function is the internal implementation of verify_cell_kzg_proof_batch.
+    Specifically, given a set of tuples (``row_index``, ``column_index``, ``evals``, ``proof``), for every tuple it
+    verifies ``proof`` which shows that ``evals`` are actually the evaluations of the polynomial behind the
+    corresponding commitment.
+
+    The value ``row_index`` points to a commitment in the array ``row_commitments``, while the value ``column_index``
```

Comment:
Another nit: maybe we can make clear what "points to" means here: "row_index points to a commitment in the array row_commitments, i.e., the relevant commitment is row_commitments[row_index]".